docs/user-guide/app-details/ephemeral-containers.md

```diff
@@ -0,0 +1,81 @@
+# Using Ephemeral Containers on Devtron
+
+## Introduction
+
+Ephemeral container is a temporary disposable tool that helps you fix issues in your application containers. It provides a gateway (terminal) to perform process-level and OS-level debugging, without impacting the target container and its resources.
```

Comment:
Definition is kinda incorrect / not clear. Please check the [official docs](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/), you can pick a few points from uses and what is section